rmltraintfbbox/rmltraintfbbox/validation/stats.py

```diff
@@ -53,11 +54,12 @@ def load_from_dump(cls, dump_path):
         self.stats = {}
         return self
 
-    def add_single_result(self, output, inference_time, bbox, centroid, image_size=None, distance=None):
+    def add_single_result(self, output, true_shape, inference_time, bbox, centroid, image_size=None, distance=None):
         """
         Add single inference result to the evaluation.
-        :param output: the parsed output from a BoundingBoxModel inference.
-        :param inference_time: the inference time from a BoundingBoxModel inference.
+        :param output: the parsed output from a detection model inference.
```

Comment:
we use Google style docstrings for everything in the codebase (https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings), and Sphinx supports them via Napoleon (https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html). would recommend switching back to google style for consistency

Reply:
This is my bad, I wrote this code and I'm used to reST style. I used it for all of starfish, and a bunch of docstrings in all the train plugins I wrote as well. I can go back and change them all later, although I don't think we should block this PR on that.

Reply:
I've never looked at the docstrings in those before, didn't realize. Not a huge deal, I just find the Google style ones easier to read. Resolving for now